### DISCUSSION TITLE
allow whitelisting for choices 3-6

### DISCUSSION THREAD

**Discussion by Neeeap** [ThioJoe/YT-Spammer-Purge#859] [Ideas]
I feel like choices 3-6 are useless right now, but maybe allowing whitelisting may help. It still prevents abuse and theres no whitelist limit, right? We can use this to lower false positives on a certain channel and use the sensitive smart to catch more spammers.

**Comment by ThioJoe** [comment]
I believe whitelisting should apply no matter what filter mode you're using, unless that has not been your experience
